### ISSUE TITLE
Broken HTML escaping host-meta check

### ISSUE THREAD

**Issue by nupplaphil** [friendica/friendica#6350]
Wrong escaping of the host-meta check in the admin-overview.
I'm using quote for expected behavior/right escaping and code for actual behavior/wrong escaping

### Expected behavior

> [https://friendica.philipp.info/.well-known/host-meta](https://friendica.philipp.info/.well-known/host-meta) is not reachable on your system. This is a severe configuration issue that prevents server to server communication. See [the installation page](https://friendica.philipp.info/help/Install) for help.

### Actual behavior

```
<a href="https://friendica.philipp.info/.well-known/host-meta">https://friendica.philipp.info/.well-known/host-meta</a> is not reachable on your system. This is a severe configuration issue that prevents server to server communication. See <a href="https://friendica.philipp.info/help/Install">the installation page</a> for help.
```

### Steps to reproduce the problem

Let the host-meta check fail

### Friendica version you encountered the problem

2018.12-rc

### Friendica source (git, zip)

git

**Comment by MrPetovan** [comment]
Part of #6208

**Comment by MrPetovan** [comment]
Closed via #6351
